### PR TITLE
feat: New Module `avm/res/load-test-service/load-test`

### PR DESCRIPTION
**Pull request by sebassem**
## Description

New module : Load-test-service.load-test

## Pipeline Reference

| Pipeline |
| -------- |
|     [avm.res.load-test-service.load-test](https://github.com/sebassem/bicep-registry-modules/actions/workflows/avm.res.load-test-service.load-test.yml)     |

## Type of Change

- [ ] Update to CI Environment or utlities (Non-module effecting changes)
- [ ] Azure Verified Module updates:
  - [ ] Bugfix containing backwards compatible bug fixes, and I have NOT bumped the MAJOR or MINOR version in `version.json`:
    - [ ] Someone has opened a bug report issue, and I have included "Closes #{bug_report_issue_number}" in the PR description.
    - [ ] The bug was found by the module author, and no one has opened an issue to report it yet.
  - [ ] Feature update backwards compatible feature updates, and I have bumped the MINOR version in `version.json`.
  - [ ] Breaking changes and I have bumped the MAJOR version in `version.json`.
  - [ ] Update to documentation

## Checklist

- [X] I'm sure there are no other open Pull Requests for the same update/change
- [X] I have run `Set-AVMModule` locally to generate the supporting module files.
- [X] My corresponding pipelines / checks run clean and green without any errors or warnings

### CODE REVIEW

**Review by AlexanderSehr — CHANGES_REQUESTED** [review]
Hey @matebarabas, is this module already onboarded to the MAR file?

**Comment by matebarabas** [comment]
> Hey @matebarabas, is this module already onboarded to the MAR file?

Hi @AlexanderSehr, yes, this one has already been onboarded to MAR.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -68,7 +68,7 @@
 /avm/res/key-vault/vault/ @Azure/avm-res-keyvault-vault-module-owners-bicep @Azure/avm-core-team-technical-bicep
 /avm/res/kubernetes-configuration/extension/ @Azure/avm-res-kubernetesconfiguration-extension-module-owners-bicep @Azure/avm-core-team-technical-bicep
 /avm/res/kubernetes-configuration/flux-configuration/ @Azure/avm-res-kubernetesconfiguration-fluxconfiguration-module-owners-bicep @Azure/avm-core-team-technical-bicep
-#/avm/res/load-test-service/load-test/ @Azure/avm-res-loadtestservice-loadtest-module-owners-bicep @Azure/avm-core-team-technical-bicep
+/avm/res/load-test-service/load-test/ @Azure/avm-res-loadtestservice-loadtest-module-owners-bicep @Azure/avm-core-team-technical-bicep
```
Comment (AlexanderSehr) [inline .github/CODEOWNERS:71]: Soft block until team is configured & approved

File: avm/res/load-test-service/load-test/tests/e2e/user-assigned-cmk-encryption/main.test.bicep [hunk 2]
```diff
@@ -0,0 +1,69 @@
+targetScope = 'subscription'
+
+metadata name = 'Using Customer-Managed-Keys with System-Assigned identity'
+metadata description = 'This instance deploys the module using Customer-Managed-Keys using a System-Assigned Identity. This required the service to be deployed twice, once as a pre-requisite to create the System-Assigned Identity, and once to use it for accessing the Customer-Managed-Key secret.'
```
Comment (AlexanderSehr) [inline avm/res/load-test-service/load-test/tests/e2e/user-assigned-cmk-encryption/main.test.bicep:4]: That would be nice, but this test does not use a system-assigned identity ;)

File: avm/res/load-test-service/load-test/tests/e2e/waf-aligned/main.test.bicep [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    lock: {
+      kind: 'None'
+    }
+    tags: {
+      'hidden-title': 'This is visible in the resource name'
+      Environment: 'Non-Prod'
+      Role: 'DeploymentValidation'
+    }
+    managedIdentities: {
+      systemAssigned: true
```
Comment (AlexanderSehr) [inline avm/res/load-test-service/load-test/tests/e2e/waf-aligned/main.test.bicep:57]: Required for WAF-alignment?

File: avm/res/load-test-service/load-test/tests/e2e/waf-aligned/main.test.bicep [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+module testDeployment '../../../main.bicep' = {
+  scope: resourceGroup
+  name: '${uniqueString(deployment().name, resourceLocation)}-test-${serviceShort}'
+  params: {
+    name: '${namePrefix}${serviceShort}001'
+    location: resourceLocation
+    enableTelemetry: enableTelemetry
+    lock: {
+      kind: 'None'
+    }
```
Comment (AlexanderSehr) [inline avm/res/load-test-service/load-test/tests/e2e/waf-aligned/main.test.bicep:50]:
```suggestion
```

File: .github/CODEOWNERS (hunk 1 above)
Comment (sebassem) [reply .github/CODEOWNERS:71]: Teams are created now

Comment (AlexanderSehr) [reply .github/CODEOWNERS:71]: I hope that means including adding them in the team hierachy ;) If so, we just need to wait for @matebarabas to approve

**Review by AlexanderSehr — APPROVED** [review]

**Pull request merged**